packages/webview_flutter/webview_flutter_wkwebview/pigeons/web_kit.dart

```diff
@@ -376,6 +389,7 @@ abstract class WKWebViewHostApi {
 
   void loadFlutterAsset(int instanceId, String key);
 
+  @ObjCSelector('webViewWithInstanceIdCanGoBack:')
```

Comment:
Sorry, I missed this before but was reminded by a recent discussion in a code cleanup issue: the ObjC name for all of these should actually be `...WithInstanceIdentifier...`. `Id` is incorrect for ObjC, and while `ID` is okay, Apple uses `Identifier` in their APIs and we should be consistent with that.

But I was also thinking as I looked at these that there's not much potential for confusion about what identifier we're referring to in the context of this class IMO, so we could probably just use `Identifier` instead of `InstanceIdentifier` everywhere. (The param can still be named `instanceId` which will make things even more obvious at the declaration point.)

Reply:
I was considering using `identifier` as well, but I noticed this made the param names inconsistent. I'm fine with either, though. I'll wait for feedback from @jmagman / @cyanglaz as well.

Reply:
`Identifier` sgtm

Reply:
Identifier +1